sqlglot/generator.py

```diff
@@ -363,8 +363,12 @@ def columndef_sql(self, expression: exp.ColumnDef) -> str:
         constraints = self.expressions(expression, key="constraints", sep=" ", flat=True)
         exists = "IF NOT EXISTS " if expression.args.get("exists") else ""
 
+        if not kind and not constraints:
```

Comment:
can you make this just return f"{exists}{column}{kind}{constraint}"

and then above add the appropriate spaces, that way we can return all these if branches

Reply:
Updated